ferry/database/stage.py

```diff
@@ -45,6 +45,8 @@ def stage(tables: dict[str, pd.DataFrame], database: Database):
             )
         # create in-memory buffer for DataFrame
         buffer = StringIO()
+        staged_tables[table.name] = staged_tables[table.name].replace(
+            {r'\r': '', r'\n': ' '}, regex=True)
```

Comment:
This looks slightly suspicious to me. Where are the extraneous line breaks coming from and what happens if we remove them?

Reply:
This code actually also works without removing the occurrences of `\n`. Would it be better if I made that change?

This is the error I was getting earlier:

`psycopg2.errors.BadCopyFileFormat: literal carriage return found in data`

I can try to look more into the cause of the occurrences of `\r`, but as of right now, I'm not sure where they are coming from and if we are losing any information by removing them.